write.go

```diff
@@ -156,6 +156,7 @@ func (b *IndexBuilder) Write(out io.Writer) error {
 	}, &toc.metaData, w); err != nil {
 		return err
 	}
+	b.repo.Symbols = len(b.runeDocSections) > 0
```

Comment:
if this is how we set the value, why do we need to write it? For the `-incremental` support we can just rely on HashOptions?